Documentation/operations/upgrade.rst

```diff
@@ -316,6 +316,10 @@ Helm Options
 ~~~~~~~~~~~~
 * The Helm option ``clustermesh.enableMCSAPISupport`` has been deprecated in favor of ``clustermesh.mcsapi.enabled``
   and will be removed in Cilium 1.20.
+* The Helm option ``clustermesh.config.clusters`` now support a new format based on a dict
```

Comment:
Do we have a plan to deprecate the old way, or is the plan to keep both methods indefinitely?